### PR TITLE
Update pg gem to 1.2.3

### PR DESCRIPTION
**Pull request by jashaik**
Signed-off-by: jan shahid shaik <jashaik@progress.com>

### Description

Update to the latest pg gem in our tooling

### Issues Resolved
Updated the pg gem to 1.2.3

### Check List

- [ ] New functionality includes tests
- [ ] All buildkite tests pass
- [ ] Full omnibus build and tests in buildkite pass
- [ ] All commits have been signed-off for the Developer Certificate of Origin. See <https://github.com/chef/chef/blob/master/CONTRIBUTING.md#developer-certification-of-origin-dco>
- [ ] PR title is a worthy inclusion in the CHANGELOG

### CODE REVIEW

**Comment by jashaik** [comment]
Pipeline Information-
Adhoc - https://buildkite.com/chef/chef-chef-server-master-omnibus-adhoc/builds/2905
Umbrella - https://buildkite.com/chef/chef-umbrella-master-chef-server/builds/566

**Comment by jashaik** [comment]
chef-server-ctl commands are tested on both postgres 9.6 and 13.3 versions
@PrajaktaPurohit @tas50

**Comment by PrajaktaPurohit** [comment]
LGTM as long as we have tested the ctl commands with both postgres 9.6 and 13.3 on upgrading the pg gem.

File: src/chef-server-ctl/chef-server-ctl.gemspec [hunk 1]
```diff
@@ -52,7 +51,7 @@ Gem::Specification.new do |spec|
   # tools we bundle in the chef-server install and include here so we can have a single Gemfile.lock
   # for the overall chef-server "app"
   spec.add_runtime_dependency "knife-ec-backup"
-  spec.add_runtime_dependency "chef_fixie"
+  spec.add_runtime_dependency "chef_fixie_shahid" #Note: replace with chef_fixie after fixing the chef_fixie pipelines
```
Comment (tas50) [inline src/chef-server-ctl/chef-server-ctl.gemspec:54]: 1.0 is out so you can make this a dep on >= 1.0 and we should be good

Comment (jashaik) [reply src/chef-server-ctl/chef-server-ctl.gemspec]: updated

**Comment by jashaik** [comment]
> LGTM as long as we have tested the ctl commands with both postgres 9.6 and 13.3 on upgrading the pg gem.

Yes Prajakta, Tested with both 9.6 and 13.3

**Review by PrajaktaPurohit — APPROVED** [review]
Looks great! thank you for all the testing on this!

**Pull request merged**